pegasus/src/forms.rb

```diff
@@ -1,34 +1,8 @@
-require 'cdo/poste'
 require 'cdo/regexp'
 require src_dir 'database'
+require File.join(CDO.root_dir, 'lib/forms/pegasus_form_errors')
```

Comment:
FYI I subsequently changed it to `require lib_dir 'forms/pegasus_form_errors'` after some PR feedback [here](https://github.com/code-dot-org/code-dot-org/pull/14781/commits/bd7d78c8940c9ccc8421c500230f1802458c8e35#diff-13e2eb4cfaa478589fe52139d7623d0aL1). This is cleaner.